static/js/src/advantage/auto-renewal.js

```diff
@@ -0,0 +1,45 @@
+import { setAutoRenewal } from "./contracts-api.js";
+
+const autoRenewalButton = document.querySelector(".js-reveal-renewal-options");
+const chevron = autoRenewalButton.querySelector(".p-icon--contextual-menu");
+const autoRenewalSection = document.getElementById("renewal-options");
+
+function toggleAutoRenewalSection() {
+  chevron.classList.toggle("u-rotate");
+  autoRenewalSection.classList.toggle("u-hide");
+
+  if (autoRenewalButton.getAttribute("aria-expanded") === "true") {
+    autoRenewalButton.setAttribute("aria-expanded", "false");
+    autoRenewalSection.setAttribute("aria-hidden", "true");
+  } else {
+    autoRenewalButton.setAttribute("aria-expanded", "true");
+    autoRenewalSection.setAttribute("aria-hidden", "false");
+  }
+}
+
+function confirmChanges() {
+  const enabledRadio = document.getElementById("auto-renewal-on");
+  const subscriptionId = this.dataset.subscriptionId;
+
+  setAutoRenewal(subscriptionId, enabledRadio.checked).catch((error) => {
+    console.error(error);
```

Comment:
Can this push an error to raven so we see it in sentry.

Reply:
Sentry logging seem to be done in the flask backend so I've added a skeleton there